test/integrations/Callouts.test.js

```diff
@@ -0,0 +1,37 @@
+import 'babel-polyfill';
+import integration from '../helpers/integration';
+
+const { it, isIOS } = integration;
+
+describe('Callouts', () => {
+  // For some reason appium can't see if the marker is opened or not
+  it('should show custom callouts', function* callback(driver, done) {
+    yield driver.elementById('Custom Callouts').click();
+
+    yield driver.elementById('Tap on markers to see different callouts');
+
+    // Click on the first callout
+    // It shows the callout
+    if (isIOS) {
+      yield driver.elementById('This is a title').click();
+    } else {
+      yield driver.elementByXPath(
```

Comment:
It was just an example, I could not compile anymore android, should not be needed